### ISSUE TITLE
[Bug]: Font Awesome Icons not appearing in polybar

### ISSUE THREAD

**Issue by EricPezzulo** [polybar/polybar#2593]
### Checklist

- [X] I have read the appropriate section in the [contributing guidelines](https://github.com/polybar/polybar/blob/master/CONTRIBUTING.md)
- [X] I believe this issue is a problem with polybar itself and not a misconfiguration on my part
- [X] I have searched for other open and closed [issues](https://github.com/polybar/polybar/issues?q=is%3Aissue) that may have already reported this problem
- [X] I have checked the [known issues](https://github.com/polybar/polybar/wiki/Known-Issues) page for this problem.
- [X] I have followed the [debugging guide](https://github.com/polybar/polybar/wiki/Debugging-your-Config) to narrow down the problem to a minimal config.

### Steps to reproduce

I updated my arch Linux desktop and all the icons in my poly bar disappeared.

### Minimal config

```dosini
[bar/top]
font-0 = fixed:pixelsize=12;2
font-1 = unifont:fontformat=truetype:size=12:antialias=false;2
font-2 = siji:pixelsize=12;2
font-3 = Font Awesome 5 Free:pixelsize=12;2
font-4 = Font Awesome 5 Free Solid:pixelsize=12;2
font-5 = Font Awesome 5 Brands:pixelsize=12;2
font-6 = Monospace:style=Regular:size=10;2
```

### Polybar log

```text
no errors
```

### Expected behavior

Icons should appear in the poly bar.

### Actual behavior

There are no icons in the poly bar

### Window Manager and Version

i3-gaps

### Linux Distribution

Arch Linux

### Polybar version

```text
polybar 3.5.7
```

**Comment by patrick96** [comment]
Please provide the full config and full log.

**Comment by Mathie-Josh** [comment]
Not the original poster, but having the same issue. Here are my files:

run log: https://pastebin.com/atG8WzA6
Config: https://pastebin.com/y4TBkDuy

The run log seems to show that the fount is loaded without issue. Could FA have changed which icons are actually showing?

**Comment by patrick96** [comment]
It didn't actually load the correct fonts:

>   5 polybar|notice:  Loaded font "Font Awesome 5 Free:style=Solid:pixelsize=13" (name=Noto Sans, offset=1, file=/usr/share/fonts/noto/NotoSans-Regular.ttf)                   

The text in parenthesis shows the actual font file that was loaded.

**Comment by Mathie-Josh** [comment]
You are correct, sorry about that. I also noticed that for me, my version of Font Awesome has actually changed. Changing this to reference the correct version (in my case "Font Awesome 6") Fixed most of the issues for me, (now just need to fix my sound and bl bars). Thank you @patrick96 for looking into what I shared! @EricPezzulo check which version of Font awesome you have that might be your issue as well.
